src/api/controllers/url.js

```diff
@@ -0,0 +1,70 @@
+const shortid = require("shortid");
+const Url = require("../../models/Url");
+const baseUrl = "http://localhost:3000";
+const utils = require("../../Utils/utils");
+const { isValidObjectId } = require("../../Utils/validate");
+
+const shorten = async (req, res) => {
+  try {
+    const { longUrl } = req.body;
+    const urlCode = shortid.generate();
+
+    if (!utils.validateUrl(longUrl)) {
+      return res.status(401).json({ message: "Invalid Long URL" });
+    }
+    let url = await Url.findOne({ longUrl });
+    if (!url) {
+      const shortUrl = baseUrl + "/" + urlCode;
+      url = new Url({
+        longUrl,
+        shortUrl,
+        urlCode,
+        date: new Date(),
+      });
+      url = await url.save();
+      return res.status(201).json(url);
+    }
```

Comment:
Remove this return statement, it is redundant, the return statement on line 27 covers it.